sdf/compositions/hub.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: Publishing
+description: Guide to publishing packages to the InfinyOn Hub.
+sidebar_position: 30
+---
+
+# Introduction
+
+The InfinyOn Hub is cloud infrastructure for sharing and discovering SDF package.  Please see [Hub] for more information.
```

Comment:
InfinyOn Hub is the cloud infrastructure for sharing and discovering SDF packages.